<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1081</width>
      <height>561</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>광고 생성</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1061</width>
        <height>521</height>
       </rect>
      </property>
      <property name="title">
       <string>광고 생성</string>
      </property>
      <widget class="QPushButton" name="search_add_btn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>쇼핑 검색 광고</string>
       </property>
      </widget>
      <widget class="QPushButton" name="link_add_btn">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>20</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>파워링크 광고</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="info_groupbox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>331</width>
         <height>111</height>
        </rect>
       </property>
       <property name="title">
        <string>쇼핑검색 광고 기본정보</string>
       </property>
       <widget class="QLabel" name="label1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>광고 캠페인 이름</string>
        </property>
       </widget>
       <widget class="QLabel" name="search_label1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>네이버쇼핑 상품ID</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>쇼핑몰 주소</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>20</y>
          <width>191</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>50</y>
          <width>191</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>80</y>
          <width>191</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="link_label1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>광고 그룹 이름</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="material_groupbox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>331</width>
         <height>121</height>
        </rect>
       </property>
       <property name="title">
        <string>확장소재</string>
       </property>
       <widget class="QLabel" name="search_label2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>확장소재 문구 1 (10자)</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>311</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="search_label3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>확장소재 문구 2 (30자)</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>311</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="link_label2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>광고 제목</string>
        </property>
       </widget>
       <widget class="QLabel" name="link_label3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>광고 설명 (최소 20자)</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>310</y>
         <width>331</width>
         <height>201</height>
        </rect>
       </property>
       <property name="title">
        <string>결과</string>
       </property>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>311</width>
          <height>171</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>20</y>
         <width>691</width>
         <height>451</height>
        </rect>
       </property>
       <property name="title">
        <string>희망 키워드</string>
       </property>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>수동 추가</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>30</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>엑셀 추가</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>노출용 상품명</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="expose_textbox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>60</y>
          <width>571</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>{희망키워드}</string>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>671</width>
          <height>341</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>희망키워드</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>노출용 상품명</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>480</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>광고 생성 시작</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>930</x>
         <y>480</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>광고 키워드 추가</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>광고 관리</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
